store/sqlstore/channel_store.go

```diff
@@ -2251,6 +2251,35 @@ func (s SqlChannelStore) GetAllChannelsForExportAfter(limit int, afterId string)
 	})
 }
 
+func (s SqlChannelStore) GetAllDirectChannelsForExportAfter(limit int, afterId string) store.StoreChannel {
+	return store.Do(func(result *store.StoreResult) {
+		var data []*model.DirectChannelForExport
+		if _, err := s.GetReplica().Select(&data, `
+			SELECT
+				Channels.*,
+				GROUP_CONCAT(Users.Username SEPARATOR ',') AS Usernames
```

Comment:
re: `GROUP_CONCAT`, I'm inclined to just split this into separate SQL queries, fetching all the channels first, and then a second query doing `WHERE ChannelId IN (...)` to fetch the channel members. Performance is slightly poorer (always better to do more work in the database), but I suspect the simpler algorithm is the place to start and we can optimize later.

While you're in here, I'd propose dispensing with the `Usernames` field and just populate `Members` directly instead of requiring the client to do this/handle some delimiter syntax.